modules/meeting/app/components/meeting_agenda_items/list_component.rb

```diff
@@ -42,7 +42,8 @@ def initialize(meeting:, form_hidden: true, form_type: :simple)
 
     def call
       component_wrapper(data: wrapper_data_attributes) do
-        render(Primer::Beta::BorderBox.new) do |border_box|
+        # making the container of items position-relative
```

Comment:
The comment is not really explanatory. The fact that `position: :relative` is making the container position-relative is kind of self-explanatory. It would be better to explain why this change is necessary, so which components are involved (position of action menu dropdown), with which steps (having to scroll) and what browser (Firefox). Ideally you'd also link the ticket.